packages/@webex/plugin-meetings/src/reachability/index.ts

```diff
@@ -291,6 +292,7 @@ export default class Reachability {
           elapsed
         );
         this.setLatencyAndClose(peerConnection, elapsed);
+        this.setClientMediaIPs(peerConnection, e.candidate.address);
```

Comment:
it's better to call this before `this.setLatencyAndClose()` - see my other comments why

Reply:
switched the order